portal/factories/app.py

```diff
@@ -233,6 +233,11 @@ def configure_blueprints(app, blueprints):
 
 def configure_logging(app):  # pragma: no cover
     """Configure logging."""
+    # Avoid duplicate logging.  Multiple handlers implies logging
```

Comment:
Nice!! :star2:

Reply:
i KNOW!  about !@$ing time we found that one!